.github/workflows/license.yml

```diff
@@ -44,4 +44,14 @@ jobs:
 
     - name: Check licenses
       # We allow only "notice" type of licenses.
-      run: go run github.com/google/go-licenses check --ignore=golang.org/x --allowed_licenses=Apache-2.0,Apache-3,BSD-3-Clause,BSD-4-Clause,CC0-1.0,ISC,MIT ./...
+        run: |
+          go run github.com/google/go-licenses check \
+            # Ignore our own codebase.
+            --ignore github.com/Jigsaw-Code \
+            --ignore=golang.org/x \
+            # Missing license file, but Apache-2.0. See https://github.com/uber-go/zap/issues/1441#issuecomment-2389189961.
+            --ignore=go.uber.org/zap/exp/zapslog \
```

Comment:
Perhaps you need to pull a more recent version: https://github.com/uber-go/zap/pull/1442.

Who is pulling it?

Reply:
I think you can do go get with the module and version you want and that should update the go.mod and sum. Have you tried that?

If you need to make them direct dependencies, you can create a dummy go files (e.g. `dependencies_hack.go`) and add them there, like we do with `tools.go`.

Reply:
Actually that breaks the build. I'm going to have to go back to ignoring the dependency.

Reply:
It seems they had a new release on October 30. Does that fix this issue?